core/platform/models_test.py

```diff
@@ -0,0 +1,281 @@
+        expected_recommendations_models.append(recommendations_models)
+        self.assertEqual(
+            tuple(expected_recommendations_models),
+            self.registry_instance.import_models(
+                [models.NAMES.recommendations]))
+
+    def test_import_models_skill(self):
+        """Tests import_models function with skill option."""
+        from core.storage.skill import gae_models as skill_models
+        expected_skills_models = [skill_models]
+        self.assertEqual(
+            tuple(expected_skills_models),
+            self.registry_instance.import_models([models.NAMES.skill]))
+
+    def test_import_models_statistics(self):
+        """Tests import_models function with statistics option."""
+        from core.storage.statistics import gae_models as statistics_models # pylint: disable=line-too-long
+        expected_statistics_models = [statistics_models]
+        self.assertEqual(
+            tuple(expected_statistics_models),
+            self.registry_instance.import_models([models.NAMES.statistics]))
+
+    def test_import_models_story(self):
+        """Tests import_models function with story option."""
+        from core.storage.story import gae_models as story_models
+        expected_story_models = [story_models]
+        self.assertEqual(
+            tuple(expected_story_models),
+            self.registry_instance.import_models([models.NAMES.story]))
+
+    def test_import_models_suggestion(self):
+        """Tests import_models function with suggestion option."""
+        from core.storage.suggestion import gae_models as suggestion_models
+        expected_suggestion_models = [suggestion_models]
+        self.assertEqual(
+            tuple(expected_suggestion_models),
+            self.registry_instance.import_models([models.NAMES.suggestion]))
+
+    def test_import_models_topic(self):
+        """Tests import_models function with topic option."""
+        from core.storage.topic import gae_models as topic_models
+        expected_topic_models = [topic_models]
+        self.assertEqual(
+            tuple(expected_topic_models),
+            self.registry_instance.import_models([models.NAMES.topic]))
+
+    def test_import_models_user(self):
+        """Tests import_models function with user option."""
+        from core.storage.user import gae_models as user_models
+        expected_user_models = [user_models]
+        self.assertEqual(
+            tuple(expected_user_models),
+            self.registry_instance.import_models([models.NAMES.user]))
+
+    def test_import_models_invalid(self):
+        """Tests import_models function with an invalid option."""
+        with self.assertRaises(Exception):
+            self.registry_instance.import_models([''])
+
+    def test_import_current_user_services(self):
+        """Tests import current user services function."""
+        from core.platform.users import gae_current_user_services as current_user_services  # pylint: disable=line-too-long
```

Comment:
Here and below, I don't think you really need the aliasing -- just compare to gae_current_user_services directly.